src/components/Foofer.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { useTodos } from '../utils/TodoContext';
+import classNames from 'classnames';
+import { Status } from '../types/Status';
+import { ErrText } from '../types/ErrText';
+
+export const Foofer: React.FC = () => {
+  const { todos, status, setStatus, deleteTodo, setErrMessage } = useTodos();
+  const itemsLeft = todos.filter(el => !el.completed).length;
+  const completedTodos = todos.filter(todo => todo.completed);
+
+  const clearCompleted = async () => {
+    try {
+      await Promise.all(completedTodos.map(todo => deleteTodo(todo.id)));
```

Comment:
This Promise all can cause test failure, please try to use just for loop and delete every todo individually.